keystone_api/main/settings.py

```diff
@@ -146,6 +146,13 @@
     'DEFAULT_SCHEMA_CLASS': 'drf_spectacular.openapi.AutoSchema',
 }
 
+SIMPLE_JWT = {
+    'ACCESS_TOKEN_LIFETIME': timedelta(minutes=60)
+}
+
+if DEBUG:  # Disable the API GUI if not in debug mode
```

Comment:
This `if DEBUG` block was removed in earlier work. It looks like it was added back in here by accident.